Source/Core/DolphinWX/FrameTools.cpp

```diff
@@ -1173,6 +1176,21 @@ void CFrame::OnMemcard(wxCommandEvent& WXUNUSED(event))
   HotkeyManagerEmu::Enable(true);
 }
 
+void CFrame::OnLoadGameCubeBIOSJAP(wxCommandEvent& WXUNUSED(event))
```

Comment:
You can simply avoid naming the parameter here and below to avoid any possible unused warning and to reduce visual noise.
i.e. `void CFrame::OnLoadGameCubeBIOSJAP(wxCommandEvent&)`